easybuild/easyconfigs/p/Paraview/Paraview-4.3.1-intel-2015a-mpi.eb

```diff
@@ -0,0 +1,51 @@
+easyblock = 'CMakeMake'
+
+name = 'Paraview'
+version = '4.3.1'
+versionsuffix = '-mpi'
```

Comment:
well, if Python is a dep (and other deps also use Python), it's good practice to include `-Python-<version>` in the versionsuffix